sdk/react-native-expo/src/ReactNativeCacheStore.ts

```diff
@@ -0,0 +1,26 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { DVCStorage } from '@devcycle/js-client-sdk'
+
+export class ReactNativeStore implements DVCStorage {
```

Comment:
if these are copy pasted from the other SDK maybe just symlink them?

Reply:
<img width="1177" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/98763537/fb550961-a62a-40de-8b4a-2051ea8e17f1">
when I symlink the files I run into this error. do you know how I can update the tsconfig to allow this? I've tried asking chatGPT and this was suggested but didn't fix it

> Review the tsconfig.json file in the react-native-expo package and ensure that it is correctly configured. Make sure that the rootDir and baseUrl settings are set to the appropriate paths. The baseUrl should point to the root of the react-native-expo package, and rootDir should encompass all source files.